ocfweb/api/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.conf.urls import url
+
+from ocfweb.api import api
+
+urlpatterns = [
+    url(r'sign\.json', api.sign_text, name='sign-json'),
```

Comment:
i'd probably remove the `.json`, that's pretty non-standard for APIs and we wouldn't include it if we e.g. expanded the API to include vhost mail things